src/bin/miri.rs

```diff
@@ -384,10 +384,14 @@ fn main() {
                     miri_config.tag_raw = true;
                 }
                 "-Zmiri-strict-provenance" => {
-                    miri_config.strict_provenance = true;
+                    miri_config.provenance_mode = ProvenanceMode::Strict;
                     miri_config.tag_raw = true;
                     miri_config.check_number_validity = true;
                 }
+                "-Zmiri-permissive-provenance" => {
+                    miri_config.provenance_mode = ProvenanceMode::Permissive;
+                    miri_config.tag_raw = true;
+                }
```

Comment:
Please add this to the README (keep the list of flags alphabetically sorted), and make sure to state there that this is currently incompatible with Stacked Borrows.